Lower required Nvidia GPU architecture

njroussel opened the pull request:
This PR fixes #182.
It updates the `drjit` submodule where we now properly check if asynchronous memory allocations are supported. It also modifies the `.ptx` generation for custom shapes so as to be supported by older Nvidia GPU architectures.

(Very few changes, PR opened to run the CI)

wjakob commented:
LGTM!

Speierers reviewed (commented):
🚀

wjakob commented:
One quick question: why new EXR files? Is it actually related to the change in minimum SM version? (that would seem suspect)

njroussel commented:
No, they are not related. I've removed that change and also rebased.

(The `resources/data` submodule is lagging on `master` I'll do that separately.)
